lib/matplotlib/_cm_listed.py

```diff
@@ -2057,6 +2057,779 @@
+    [0.19578, 0.26567, 0.097702],
+    [0.20067, 0.2729, 0.099539],
+    [0.20564, 0.28016, 0.10144],
+    [0.21062, 0.28744, 0.10342],
+    [0.21565, 0.29475, 0.10534],
+    [0.22072, 0.30207, 0.10737],
+    [0.22579, 0.30942, 0.10942],
+    [0.23087, 0.31675, 0.11146],
+    [0.236, 0.32407, 0.11354],
+    [0.24112, 0.3314, 0.11563],
+    [0.24625, 0.33874, 0.11774],
+    [0.25142, 0.34605, 0.11988],
+    [0.25656, 0.35337, 0.12202],
+    [0.26171, 0.36065, 0.12422],
+    [0.26686, 0.36793, 0.12645],
+    [0.272, 0.37519, 0.12865],
+    [0.27717, 0.38242, 0.13092],
+    [0.28231, 0.38964, 0.13316],
+    [0.28741, 0.39682, 0.13541],
+    [0.29253, 0.40398, 0.13773],
+    [0.29763, 0.41111, 0.13998],
+    [0.30271, 0.4182, 0.14232],
+    [0.30778, 0.42527, 0.14466],
+    [0.31283, 0.43231, 0.14699],
+    [0.31787, 0.43929, 0.14937],
+    [0.32289, 0.44625, 0.15173],
+    [0.32787, 0.45318, 0.15414],
+    [0.33286, 0.46006, 0.1566],
+    [0.33781, 0.46693, 0.15904],
+    [0.34276, 0.47374, 0.16155],
+    [0.34769, 0.48054, 0.16407],
+    [0.3526, 0.48733, 0.16661],
+    [0.35753, 0.4941, 0.16923],
+    [0.36245, 0.50086, 0.17185],
+    [0.36738, 0.50764, 0.17458],
+    [0.37234, 0.51443, 0.17738],
+    [0.37735, 0.52125, 0.18022],
+    [0.38238, 0.52812, 0.18318],
+    [0.38746, 0.53505, 0.18626],
+    [0.39261, 0.54204, 0.18942],
+    [0.39783, 0.54911, 0.19272],
+    [0.40311, 0.55624, 0.19616],
+    [0.40846, 0.56348, 0.1997],
+    [0.4139, 0.57078, 0.20345],
+    [0.41942, 0.57819, 0.20734],
+    [0.42503, 0.5857, 0.2114],
+    [0.43071, 0.59329, 0.21565],
+    [0.43649, 0.60098, 0.22009],
+    [0.44237, 0.60878, 0.2247],
+    [0.44833, 0.61667, 0.22956],
+    [0.45439, 0.62465, 0.23468],
+    [0.46053, 0.63274, 0.23997],
+    [0.46679, 0.64092, 0.24553],
+    [0.47313, 0.64921, 0.25138],
+    [0.47959, 0.6576, 0.25745],
+    [0.48612, 0.66608, 0.26382],
+    [0.49277, 0.67466, 0.27047],
+    [0.49951, 0.68335, 0.2774],
+    [0.50636, 0.69213, 0.28464],
+    [0.51331, 0.70101, 0.2922],
+    [0.52035, 0.70998, 0.30008],
+    [0.5275, 0.71905, 0.30828],
+    [0.53474, 0.72821, 0.31682],
+    [0.54207, 0.73747, 0.32567],
+    [0.5495, 0.74682, 0.33491],
+    [0.55702, 0.75625, 0.34443],
+    [0.56461, 0.76577, 0.35434],
+    [0.5723, 0.77537, 0.36457],
+    [0.58006, 0.78506, 0.37515],
+    [0.58789, 0.79482, 0.38607],
+    [0.59581, 0.80465, 0.39734],
+    [0.60379, 0.81455, 0.40894],
+    [0.61182, 0.82453, 0.42086],
+    [0.61991, 0.83457, 0.43311],
+    [0.62805, 0.84467, 0.44566],
+    [0.63623, 0.85482, 0.45852],
+    [0.64445, 0.86503, 0.47168],
+    [0.6527, 0.8753, 0.48511],
+    [0.66099, 0.88562, 0.49882],
+    [0.6693, 0.89599, 0.51278],
+    [0.67763, 0.90641, 0.52699],
+    [0.68597, 0.91687, 0.54141],
+    [0.69432, 0.92738, 0.55605],
+    [0.70269, 0.93794, 0.5709],
+    [0.71107, 0.94855, 0.58593],
+    [0.71945, 0.9592, 0.60112],
+    [0.72782, 0.96989, 0.61646],
+    [0.7362, 0.98063, 0.63191],
+    [0.74458, 0.99141, 0.64748]]
```

Comment:
This sees fine, but are these really defined by a colourable, or do they also have a linear segmented definition?  If so, that would be much more compact....

Reply:
Hmm, I guess not, according to the originals...

Reply:
I don't know enough to dare changing from the original version, personally.